tests/unit/splinter_test.c

```diff
@@ -634,6 +634,8 @@ CTEST2(splinter, test_lookups)
  */
 CTEST2(splinter, test_splinter_print_diags)
 {
+   set_log_streams_for_tests(MSG_LEVEL_DEBUG);
```

Comment:
This call will setup the output log handles if test-case is run with `VERBOSE=7`. This allows for manual inspection of the outputs while suppressing tons of output in CI (which invokes tests using `VERBOSE=1`).